stdlib/email/src/main/ballerina/src/email/listener_endpoint.bal

```diff
@@ -164,6 +169,7 @@ public type ListenerConfig record {|
     string protocol = "IMAP";
     PopConfig|ImapConfig? protocolConfig = ();
     int pollingInterval = 60000;
+    int noOfPolls?;
```

Comment:
what is the use of setting `noOfPolls` from the user perspective?

Reply:
Is it possible to call the listener `stop()` function if we need to stop the polling.

Reply:
Suppose the user wants to handle the error path when there are no emails arriving within a certain time period, he can set that time period as `pollingInterval * noOfPolls`.

Reply:
Yes he can use the`stop()` function too but with this new field it can automatically exit the listening task like a timeout.

Reply:
Reverted the field.